<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionale</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="agentiButton">
        <property name="text">
         <string>&amp;Agenti</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clnFrnButton">
        <property name="text">
         <string>&amp;Clienti/Fornitori</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="magazzinoButton">
        <property name="text">
         <string>Magazzino</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>251</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Info</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Utente:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="db_user">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Database:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="db_name">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Stato:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="db_status">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionSetup_Connection"/>
    <addaction name="actionSetup_Table"/>
    <addaction name="actionConfigura_Azienda"/>
   </widget>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="text">
    <string>&amp;Login</string>
   </property>
  </action>
  <action name="actionSetup_Connection">
   <property name="text">
    <string>&amp;Connessione</string>
   </property>
  </action>
  <action name="actionSetup_Table">
   <property name="text">
    <string>C&amp;onfigura Tabelle</string>
   </property>
  </action>
  <action name="actionConfigura_Azienda">
   <property name="text">
    <string>Configura Azienda</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionLogin</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchLoginDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSetup_Connection</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchConnectionSetupDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSetup_Table</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchTableSetupDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>agentiButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchAgentiViewDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clnFrnButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchAnagraficaDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConfigura_Azienda</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchAziendaDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>magazzinoButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>launchMagazzinoDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>launchLoginDlg()</slot>
  <slot>launchConnectionSetupDlg()</slot>
  <slot>launchTableSetupDlg()</slot>
  <slot>launchAgentiViewDlg()</slot>
  <slot>launchAnagraficaDlg()</slot>
  <slot>launchAziendaDlg()</slot>
  <slot>launchMagazzinoDlg()</slot>
 </slots>
</ui>
